packages/flutter_markdown/test/text_selection_controls.dart

```diff
@@ -0,0 +1,316 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+
+const double _kHandleSize = 22.0;
```

Comment:
Dart constants should not use `k`, per Dart style.

Reply:
Updated.